MyLibrary/Sources/ScheduleFeature/Schedule.swift

```diff
@@ -259,6 +284,32 @@ public struct ScheduleView: View {
         }
       }
       .frame(maxWidth: .infinity, alignment: .leading)
+
+      favoriteIcon(for: session)
+        .onTapGesture {
+          send(.favoriteIconTapped(session))
+        }
+    }
+  }
+
+  @ViewBuilder
+  func favoriteIcon(for session: Session) -> some View {
```

Comment:
These logic should put in Reducer so that we can write test easily.

Reply:
I'm sorry that I can't image which part of logic to put in Reducer.
Does it mean create new `Schedule.Action` case and handle it in `Reduce`?
Or create computed property getting `Conference` in `Schedule.State`?

Now I implemented `Favorites` to use `Conference.title` as key for getting value of favorited `[Session]`.

Reply:
I moved these logic to `Schedule.swift` in d3b990178c97e4c5779249990ce492e4ff0d16c3.

Reply:
Sorry for my lack of explanation.
I moved a part of logic as far as possible, so now `State.favorites: [String: [Session]]` is used in view whether each session is favorited or not, only.